samples/amd-webpack-react/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "monaco-kusto--amd-webpack-react",
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "start": "webpack watch",
+        "build": "webpack --mode=production --node-env=production",
+        "build:dev": "webpack --mode=development",
+        "build:prod": "webpack --mode=production --node-env=production",
+        "watch": "webpack --watch",
+        "serve": "webpack serve",
+        "copy-amd-files": "copyMonacoFilesAMD ./public"
```

Comment:
```suggestion
```

Can we delete?

Reply:
No, this script must be run before the example works. See the README.md